### PR TITLE
fix(numeric): numeric fixes the issue where components disappear after being out of focus when inputting very large numbers into scientific notation.

### PR DESCRIPTION
**Pull request by shonen7**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
输入超大数字变为科学计数法时失焦后组件消失

Issue Number: N/A

## What is the new behavior?
修复输入超大数字变为科学计数法时失焦后组件消失的问题，科学计数法时，加减数值均不生效
## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/renderless/src/common/bigInt.ts [hunk 1]
```diff
@@ -186,7 +186,7 @@ export class BigIntDecimal {
       const trimRet = trimNumber(mergedValue)
       this.negative = trimRet.negative
       const numbers = trimRet.trimStr.split('.')
-      this.integer = BigInt(numbers[0])
+      this.integer = numbers[0].indexOf('e') == -1 ? BigInt(numbers[0]) : numbers[0]
```
Comment (kagol) [inline packages/renderless/src/common/bigInt.ts]:
```suggestion
      this.integer = numbers[0].indexOf('e') === -1 ? BigInt(numbers[0]) : numbers[0]
```

File: packages/renderless/src/numeric/index.ts [hunk 2]
```diff
@@ -269,7 +278,7 @@ export const handleInput =
       emitError()
 
       if (!(value === '' && props.allowEmpty)) {
-        value = state.lastInput
+        value = value.indexOf('e') == -1 ? state.lastInput : value
```
Comment (kagol) [inline packages/renderless/src/numeric/index.ts]:
```suggestion
        value = value.indexOf('e') === -1 ? state.lastInput : value
```

File: packages/renderless/src/numeric/index.ts [hunk 3]
```diff
@@ -96,6 +96,11 @@ export const increase =
     }
 
     const value = (props.mouseWheel ? state.displayValue : props.modelValue) || 0
+
+    if (value.toString().indexOf('e') !== -1) {
```
Comment (Huangyilin19) [inline packages/renderless/src/numeric/index.ts]: indexOf('') !== -1 可以用includes方法代替,更简明高效

**Review by kagol — DISMISSED** [review]

**Review by Huangyilin19 — APPROVED** [review]

File: packages/renderless/src/numeric/index.ts (hunk 3 above)
Comment (shonen7) [reply packages/renderless/src/numeric/index.ts:100]: > indexOf('') !== -1 可以用includes方法代替,更简明高效

这个组件里面查询都是indexOf('') !== -1，感觉一致性好一点

Comment (shonen7) [reply packages/renderless/src/numeric/index.ts]: > indexOf('') !== -1 可以用includes方法代替,更简明高效

已修改

**Review by kagol — APPROVED** [review]

**Pull request merged**